src/support.h

```diff
@@ -29,16 +32,44 @@ class AreaSupport {
      */
     static void generateSupportAreas(SliceDataStorage& storage, unsigned int mesh_idx, unsigned int layer_count, std::vector<Polygons>& supportAreas);
 
+    /*!
+     * Generate support bottom areas for a given mesh.
+     *
```

Comment:
The descriptions should say what these functions do to the support infill areas.